Capri.Services/IPromoterService.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Capri.Database.Entities;
+using Capri.Web.ViewModels.Promoter;
+
+namespace Capri.Services
+{
+    public interface IPromoterService
```

Comment:
Using one service to do every CRUD operation violates `Singe Responsibility Principle`.  You should split that functionalities to `IPromoterCreator`, `IPromoterUpdater`, `IPromoterGetter` `IPromoterDeleter`.